app/docker/views/services/edit/service.html

```diff
@@ -17,7 +17,7 @@
         <table class="table">
           <tbody>
             <tr>
-              <td>Name</td>
+              <td class="col-xs-6 col-sm-4 col-md-3 col-lg-3">Name</td>
```

Comment:
I suggest using tailwind classes instead. col-X classes are not really for table cells, and we're trying to deprecate the use of bootstrap

Reply:
Nice spot! I didn't notice this change - will update